webstruct/sequence_encoding.py

```diff
@@ -190,3 +191,66 @@ def classify(self, token):
 
         # regular token
         return 'token', token
+
+
+def bilou_encoder(iob_tags):
```

Comment:
function and method names should be verbs in most cases ("bilou_encode", not "bilou_encoder")